modules/Bio/EnsEMBL/Analysis/Hive/Config/TranscriptSelection.pm

```diff
@@ -0,0 +1,910 @@
+      -host   => $self->o('projection_db_server'),
+      -port   => $self->o('projection_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'rnaseq_for_layer_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_rnalayer_nr_' . $self->o('release_number'),
+      -host   => $self->o('rnaseq_for_layer_db_server'),
+      -port   => $self->o('rnaseq_for_layer_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'layering_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_layer_' . $self->o('release_number'),
+      -host   => $self->o('layering_db_server'),
+      -port   => $self->o('layering_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'utr_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_utr_' . $self->o('release_number'),
+      -host   => $self->o('utr_db_server'),
+      -port   => $self->o('utr_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'genebuilder_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_gbuild_' . $self->o('release_number'),
+      -host   => $self->o('genebuilder_db_server'),
+      -port   => $self->o('genebuilder_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'pseudogene_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_pseudo_' . $self->o('release_number'),
+      -host   => $self->o('pseudogene_db_server'),
+      -port   => $self->o('pseudogene_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'ncrna_db' => {
+      -dbname => $self->o('ncrna_db_name'),
+      -host   => $self->o('ncrna_db_server'),
+      -port   => $self->o('ncrna_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'final_geneset_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_final_' . $self->o('release_number'),
+      -host   => $self->o('final_geneset_db_server'),
+      -port   => $self->o('final_geneset_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+  };
+}
+
+sub pipeline_create_commands {
+  my ($self) = @_;
+  return [
+    # inheriting database and hive tables' creation
+    @{ $self->SUPER::pipeline_create_commands },
+
+    'mkdir -p ' . $self->o('genome_dumps'),
+  ];
+}
+
```

Comment:
I would be for removing the whole block as even if the genome_dumps didn't exist, no analysis would create the fasta file the should be in the directory.
I would be fine if only line 305 and 306 are deleted as it can make life easier if we need to add code to the method

Reply:
I deleted only line 305 and 306, we can delete the method later if needed